test/orange/data/field-layers.org.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 UT-Battelle, LLC and other Celeritas Developers.
```

Comment:
Maybe you could add a diagram/description so people can understand what this geometry is supposed to look like at a glance?